quarkus/dishmanagement/src/main/java/com/devonfw/application/dishmanagement/common/to/CategoryCto.java

```diff
@@ -0,0 +1,19 @@
+package com.devonfw.application.dishmanagement.common.to;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Composite transport object of Category
+ */
+@Getter
+@Setter
+@ToString
+public class CategoryCto {
```

Comment:
If there are no relations a CTO is pointless and shall be omitted. (https://github.com/devonfw/devon4j/blob/master/documentation/guide-eto-cto.asciidoc#cto)